common/on_action/on_game_initialized_field_of_mars.txt

```diff
@@ -1,536 +0,0 @@
-﻿on_game_initialized = {
```

Comment:
oof. this is a massive awful file. might need to diff this and see what's different from vanilla other the terrain stuff

Reply:
This file was a pain to look at and i'm surprised it just worked tbh